backend/src/domains/base.entity.ts

```diff
@@ -0,0 +1,25 @@
+import { PrimaryGeneratedColumn, Column, UpdateDateColumn, CreateDateColumn } from 'typeorm';
+
+export abstract class BaseEntity {
+    // @PrimaryGeneratedColumn()
```

Comment:
Why do you comment on this line? Probably you should leave a comment for what it can be used in the future or just delete.

Reply:
Need to be removed in the future